### DISCUSSION TITLE
Add Jake's resume theme

### DISCUSSION THREAD

**Discussion by nhuhoang0701** [rendercv/rendercv#23] [Ideas]
Great project, looks very promising! It would be nice if someone could add a theme for Jake's resume, which I mainly use for my CV and I believe it's also one of the templates students use the most!

**Comment by sinaatalay** [comment]
Thank you for the suggestion!

I looked at Jake's Resume, and it seems to be based on the `sb2nov` theme, one of the built-in themes of RenderCV. I think a CV very similar to Jake's Resume can be achieved with the `sb2nov` theme. It might be unfeasible to maintain two very similar themes.
